Update doc for how to re-run an action with another RBE instance. (#590)
bazelbuild/remote-apis-sdks: go/README.md

- An action digest formatted as `"{hash}/{size}"` can be provided directly using the `--digest` flag, or a previously downloaded action can be used with `--action_root`. The `--action_root` path should point to the `--path` of a previous `download_action` invocation. Specifically, the `--action_root` folder must contain a `cmd.textproto` and `ac.textproto` files.
- `--path` is the destination where the outputs of the action will be downloaded

#### Re-running a downloaded action remotely on a different RBE instance

If you want to download an action from an RBE instance `FOO`, and remotely 
execute it on another RBE instance `BAR` (probably because you only have the 

> permission to download actions from `FOO`, but lack of permission to don't have 
> the permission to execute an action with instance `FOO`). In this case, you want

Reviewer: ```suggestion
permission to download actions from `FOO`, but don't have 
the permission to execute that instance). In this case, you want
```
